### PR TITLE
Configure clang-format and SwiftLint

### PR DESCRIPTION
**Pull request by tido64**

### CODE REVIEW

File: ios/ReactTestApp/AppDelegate.swift [hunk 1]
```diff
@@ -1,9 +1,9 @@
-/**
- * Copyright (c) Microsoft Corporation. All rights reserved.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
+//
```
Comment (ArazAbishov) [inline ios/ReactTestApp/AppDelegate.swift]: Do we need to add this header to all files? It will be cleaner just to have the license file in the repo root.

Comment (tido64) [reply ios/ReactTestApp/AppDelegate.swift]: This is not the full license though. It just says where you can find it.

Comment (ArazAbishov) [reply ios/ReactTestApp/AppDelegate.swift]: Correct me if I am wrong, but I believe that you don't have to link the license in every source code file. I would rather prefer not to have it, as it just adds noise to files.

Comment (tido64) [reply ios/ReactTestApp/AppDelegate.swift]: It is not required but is recommended practice. If people were sent single files separated from the project, they would still know what license is applied. I’d prefer to leave it in.

**Pull request merged**